functions/to.fish

```diff
@@ -68,6 +69,22 @@ function __to_ls
   end
 end
 
+function __to_here
+    set -x currentdir (pwd)
+    set -x wholepath (ls -l ~/.tofish | grep $currentdir | cut -d':' -f 2 | cut -d' ' -f 2,4 )
```

Comment:
Use `$TO_DIR` instead of hardcoding `~/.tofish`.

`grep` will match subdirectories of `$currentdir`

I think this won't work if any of the paths contain spaces. Consider using `stat` instead of trying to parse the output of `ls`. I think you should iterate through the links in `~/.tofish` and match `(realpath $currentdir)`

Reply:
the grep to match $currentdir gets all of its subdirs, yes, then the the inner loop refines to make sure it doesn't have any of the subs, that it only matches pwd